geniesp/bpc_redcap_export_mapping.py

```diff
@@ -185,12 +185,12 @@ def configure_mafdf(mafdf, keep_samples):
     keep_mafdf = mafdf[
         mafdf['Tumor_Sample_Barcode'].isin(keep_samples.tolist())
     ]
-    if not keep_mafdf.empty:
-        fillnas = ['t_depth', 't_ref_count', 't_alt_count',
-                   'n_depth', 'n_ref_count', 'n_alt_count']
-        for col in fillnas:
-            keep_mafdf[col].loc[keep_mafdf[col] == "."] = ""
-        keep_mafdf["Validation_Status"] = ''
+    # if not keep_mafdf.empty:
```

Comment:
Why is this commented out?

Reply:
Was getting a type comparison error (int vs. string) that stopped the script.  Checking separately, the maf file for the columns listed in fillnas did not contain any instance of the "." character, so I commented out the block but couldn't figure out why this hadn't been an issue before, given it's the same maf file.  Do you think it's a python version issue (was running with 3.6.9) or something else?

Reply:
This is because in vcf's the way you define a blank space is by a `"."`, so sometimes the (at least with vcf2maf), this annotation gets propogated and I think this was something that the cBioPortal told me to do.  Bascially those NA columns shouldn't be "." but should be blank instead.

Reply:
Got it.  The issue may derive from using pandas 0.22.0 instead of >=1.0 as specified in the requirements.  I will update pandas and uncomment to see if it will run without errors.